PowerEditor/installer/nativeLang/kurdish.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
What's the change of this line?

Reply:
Changed from UTF8 with BOM -> plain UTF8 otherwise the switch to this language is not working.